### PR TITLE
Update directives.md

### PR DESCRIPTION
**Pull request by akshbhu**
*Issue #3860

*Description of changes:*
Not supporting t2.micro in amplify cli as t2.micro only works with version 1.5 and 2.3 and CLI supports version >= 6

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by brene** [comment]
@kaustavghosh06 can you also sign off?

File: docs/cli/graphql-transformer/directives.md [hunk 1]
```diff
@@ -1912,6 +1912,8 @@ The `@searchable` directive handles streaming the data of an `@model` object typ
 
 > **Note**: `@searchable` is not compatible with DataStore but you can use it with the API category. 
 
+> **Note**: `@searchable` is not compatible with  t2.micro instance as it only works with version 1.5 and 2.3 and Amplify CLI support instances with version >= 6.x.
```
Comment (kaustavghosh06) [inline docs/cli/graphql-transformer/directives.md:1915]: what's the version number we're talking out here? Elasticsearch verision?

Comment (akshbhu) [reply docs/cli/graphql-transformer/directives.md:1915]: yes, Its the Elasticsearch version

Comment (brene) [reply docs/cli/graphql-transformer/directives.md]: @akshbhu @kaustavghosh06 - just updated the copy here to be more explicit. Can you double check?

**Pull request merged**